CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 0.21.5
+
+**Commit Delta**: [Change from 0.21.4 release](https://github.com/plus3it/watchmaker/compare/0.21.4...0.21.5)
+
+**Released**: 2021.02.25
+
+**Summary**:
+
+*   ash-linux-formula
+    - Replace watch with listen to address service restarts
+    - Make state RHEL-07-040560 more resilient when the yum group info is missing
+*   scap-formula
+    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)
+*   watchmaker-salt-content
```

Comment:
We didn't actually modify the `watchmaker-salt-content` submodule. That's a separate repo. We modified the default `config.yaml`, which is part of the watchmaker repo.

```suggestion
*   Update watchmaker default `config.yaml` to use salt v2019.2.8
```